pkg/globalmanager/controllers/jointinference/jointinferenceservice.go

```diff
@@ -80,6 +83,11 @@ type Controller struct {
 	// A store of service
 	serviceLister sednav1listers.JointInferenceServiceLister
 
+	//deploymentStoreSynced returns true if the deployment store has been synced at least once.
+	deploymentStoreSynced cache.InformerSynced
+	// A store of deployment
+	deploymentsLister appslisters.DeploymentLister
```

Comment:
Now that deployment is in use as a controller, can the pod related monitoring logic be removed from the code?